torch/distributed/algorithms/ddp_comm_hooks/ddp_zero_hook.py

```diff
@@ -0,0 +1,344 @@
+from typing import Any, Callable, List, Optional
+
+import torch
+import torch.distributed as dist
+from torch.distributed.optim import ZeroRedundancyOptimizer
+from torch.distributed.optim.zero_redundancy_optimizer import _OverlapStatus
+from torch.nn.parallel.distributed import DistributedDataParallel
+from torch.distributed import distributed_c10d
```

Comment:
Do we have to explicitly import `distributed_c10d`? If the goal was to use `Backend.NCCL`, will `dist.Backend.NCCL` work?

https://pytorch.org/docs/master/distributed.html#torch.distributed.Backend

Reply:
Changed to `dist.Backend.NCCL`.